### PR TITLE
feat(sdk): added eip1559 property to statics

### PR DESCRIPTION
**Pull request by varun-bitgo**
TICKET: WP-1219

Added EIP1559 coin feature to all the supported coins and tokens in the sdk.

### CODE REVIEW

File: modules/statics/test/unit/coins.ts [hunk 1]
```diff
@@ -682,3 +682,25 @@ describe('Bulk Transaction Features', () => {
     });
   });
 });
+
+describe('Eip1559 coins', () => {
+  const eip1559Coins = [
```
Comment (margueriteblair) [inline modules/statics/test/unit/coins.ts:687]: Should also be added for `eth` and `hteth` and mainnet `avaxc` too

File: modules/statics/test/unit/coins.ts [hunk 2]
```diff
@@ -682,3 +682,28 @@ describe('Bulk Transaction Features', () => {
...
+    'avaxc',
+    'eth',
+    'hteth:bgerchv2',
+    'tavaxc',
```
Comment (margueriteblair) [inline modules/statics/test/unit/coins.ts:691]: nit: can we reorder these so `avaxc` and `tavaxc` are next to each other? prefer all the mainnet coins to be next to their testnet coins

**Review by margueriteblair — DISMISSED** [review]
lgtm

**Review by margueriteblair — DISMISSED** [review]

**Review by bitgoAaron — DISMISSED** [review]

**Review by zahin-mohammad — DISMISSED** [review]

**Review by zahin-mohammad — APPROVED** [review]

**Review by zahin-mohammad — CHANGES_REQUESTED** [review]
Looks like a bad merge conflcit.... please fix

**Review by zahin-mohammad — APPROVED** [review]

**Pull request merged**